### PR TITLE
feat!: Drop support for CUBEJS_ENABLE_TLS

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

File: packages/cubejs-backend-shared/src/env.ts [hunk 1]
```diff
@@ -8,10 +8,6 @@ const variables = {
     .default(4000)
     .required()
     .asPortNumber(),
-  tlsPort: () => get('TLS_PORT')
-    .default(4433)
-    .required()
-    .asPortNumber(),
   tls: () => get('CUBEJS_ENABLE_TLS')
```
Comment (hassankhan) [inline packages/cubejs-backend-shared/src/env.ts:11]: Should this property also be removed?

**Review by hassankhan — CHANGES_REQUESTED** [review]
PR needs to be rebased to include latest changes to `DEPRECATION.md`

File: packages/cubejs-backend-shared/src/env.ts (hunk 1 above)
Comment (ovr) [reply packages/cubejs-backend-shared/src/env.ts:11]: @hassankhan, I am still using it to detect that CUBEJS_ENABLE_TLS is used 😄

Comment (hassankhan) [reply packages/cubejs-backend-shared/src/env.ts:11]: Ahh yes, good point 😅

**Review by vasilev-alex — APPROVED** [review]

**Pull request merged**